src/client/windshaft-filtering.js

```diff
@@ -169,6 +169,7 @@ class AggregationFiltering {
         if (f.isA(ClusterAvg) || f.isA(ClusterMax) || f.isA(ClusterMin) || f.isA(ClusterMode) || f.isA(ClusterSum)) {
             let p = this._property(f.property);
             if (p) {
+                // FIXME: should we just use the cluster aggr expression?
```

Comment:
?

Reply:
thanks for the heads-up, it's just a comment for myself I forgot about; I'll manage it